Mini-MaxSum.cs

```diff
@@ -0,0 +1,46 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+using static System.Console;
+class Result
+{
+    public static void miniMaxSum(List<int> arr)
+    {
+       var min = long.MaxValue;
+       var max = 0L;
+       var sum = 0L;
+       for (int i = 0; i < 5; i++)
```

Comment:
why 5? Is the length of the array constant? Please use a length of the array instead of constant like 5 @samradhshrma

Reply:
Yes the length was given as 5 , but next time surely use lenght.